components/brave_wallet_ui/components/shared/checkbox/checkbox.style.tsx

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import styled, { css, CSSProperties } from 'styled-components'
+
+interface StyleProps {
+  selected?: boolean
+  size: 'big' | 'small'
+  disabled?: boolean
+}
+
+const getLabelProps = (p: StyleProps) => {
+  return css`
+    --checkbox-border-size: 1px;
+    --checkbox-box-size: ${p.size === 'big' ? 24 : 18}px;
+    --checkbox-label-size: ${p.size === 'big' ? 16 : 14}px;
+    --checkbox-box-spacing: ${p.size === 'big' ? 17 : 12}px;
+    --checkbox-box-color: #696FDC;
+    --checkbox-label-color: #B8B9C4;
```

Comment:
Are these available in `theme`?

Same for the bellow in this file.

Reply:
I was unable to find them using the `theme` intellisense